docs/primer_short.md

```diff
@@ -0,0 +1,146 @@
+# Test-negative design: short primer
+
+## Draft outline
+
+- What is VE
+  - Known biases with any VE estimator
+- What is a TND study
+- There are 8 kinds of people:
+  - Vaccinated vs. not
+  - Infected with the focal pathogen vs. not
+  - Seek a test vs. not
+  - Test positive vs. not
+- TND has access only to people who seek a test
+- "Actual" VE is: [insert risk ratio math]
+- TND estimator of VE is: [insert odds ratio math]
+- Note that TND estimator is unbiased only when: [assumptions]
+- Bias of TND estimator scales this way with underlying parameters: [discuss]
+
+"HLS" refers to Halloran, Longini, and Struchiner _Design and Analysis of Vaccine Studies_.
+
+## Vaccine efficacy
+
+VE is the fractional reduction in the probability of an adverse outcome due to vaccination. For purposes of a test-negative design (TND) study, we further define VE to be:
+
+- A direct effect, that is, the counterfactual reduction in risk that would occur if a single individual were vaccinated or not, and not the population-level effect of reducing transmission via a vaccination program
+- VE against infection, written $\mathrm{VE}_S$ in HLS
+- Binary: Individuals are considered vaccinated or not (e.g., people who were vaccinated shortly before the outcome of interest may be classified as "unvaccinated" in the study, because they are presumed to have not yet been protected)
+- Conditioned on some probability or amount of exposure to disease
+
+Thus:
+
+$$
+\mathrm{VE} = 1 - \frac{P[\text{infected} | \text{vaccinated}]}{P[\text{infected} | \text{unvaccinated}]}
+$$
+
+## Test-negative design
+
+A test-negative design (TND) is a study design that can be used to estimate VE. In a TND:
+
+- The population of interest are those individuals who received a diagnostic test
+- The main covariate of interest is vaccination status
+- The outcome of interest is the test result (positive or negative)
+
+The data available in a TND ultimately reduces to a 2x2 table of counts:
+
+|               | Vaccinated | Unvaccinated |
+| ------------- | ---------- | ------------ |
+| Test positive | $C_{PV}$   | $C_{PU}$     |
+| Test negative | $C_{NV}$   | $C_{NU}$     |
+
+## Mathematical model
+
+### Single-exposure model
+
+- There is only one opportunity for exposure per individuals; individuals are either exposed or not
+- Individuals are vaccinated with probability $v$
+- Individuals are exposed with probability $\varepsilon_V$ or $\varepsilon_U$
+- Exposed individuals are infected ($I$) with probability $\lambda_V$ or $\lambda_U$
+- Infected individuals seek and receive a test with probability $\mu_{V|I}$ or $\mu_{U|I}$
+  - Note that this probability represents a combination of developing symptoms, seeking healthcare, and receiving a test given that healthcare was sought
+  - $1-\mu_V/\mu_U$ is an example of VE against progression conditional on an earlier outcome, written $\mathrm{VE}_P$ in HLS
```

Comment:
The thing I struggle with separating infection->symptoms and symptoms->seeking is that these two things will always be perfectly confounded in a TND. We never see asymptomatic infecteds because we assume that anyone who is test positive and appears in the study (i.e., is "symptomatic") is in the study because they were infected.

So I guess I'm saying there's no such thing as symptom, except the thing that causes you to get entered into the study, which is circular.

I can see values both ways, pedagogically, either to say "these two Greek letters are different things, but look they always appear together in these equations, so the TND can't separate them" or my more complicated "there is no spoon"

Reply:
Update: after redoing some derivations, it seems like $\pi$ (infection -> symptoms) and $\mu$ (symptoms -> test) always go together in this math?

Reply:
I redid my notation! and math. Everything has changed.